CONTRIBUTING.md

```diff
@@ -0,0 +1,141 @@
+# Welcome #
+
+We're so glad you're thinking about contributing to this open source
+project!  If you're unsure or afraid of anything, just ask or submit
+the issue or pull request anyway.  The worst that can happen is that
+you'll be politely asked to change something.  We appreciate any sort
+of contribution, and don't want a wall of rules to get in the way of
+that.
+
+Before contributing, we encourage you to read our CONTRIBUTING policy
+(you are here), our [LICENSE](LICENSE), and our [README](README.md),
+all of which should be in this repository.
+
+## Issues ##
+
+If you want to report a bug or request a new feature, the most direct
+method is to [create an
+issue](https://github.com/cisagov/skeleton-docker/issues) in this
```

Comment:
Change _all_ references to "cisagov/skeleton-docker" in this file to "cisagov/domain-manager-api".

Reply:
Fixed that in https://github.com/cisagov/domain-manager-api/pull/194/commits/bdf4957541d689ea3583c506bc5ecf22ffe5790f